src/app/users/page.tsx

```diff
@@ -1,9 +1,73 @@
-import React from 'react'
+"use client";
+import axios from "axios";
+import { useEffect, useState } from "react";
 
-const User = () => {
-  return (
-    <div> User</div>
-  )
+interface User {
+  id: number;
+  firstName: string;
+  lastName: string;
+  email: string;
+  fatherName: string;
+  motherName: string;
+  address: string;
+  pincode: string;
+  country: string;
 }
 
-export default User
\ No newline at end of file
+const Users = () => {
+  const [userData, setUserData] = useState<User[]>([]);
+
+  const getUsers = async () => {
```

Comment:
this is very incorrect pattern of calling the api, it will create a race condition. you should either move this fucntion to useEffect or useCallback hook. read more about race conditions in react.